### PR TITLE
Sort mypy.options.PER_MODULE_OPTIONS and fix typo

### PR DESCRIPTION
**Pull request by e-kwsm**
This should not affect the current behavior.

### CODE REVIEW

**Review by gvanrossum — CHANGES_REQUESTED** [review]
Can you undo the changes to mypyc/external/googletest? Those are copied from https://github.com/google/googletest/tree/master/googletest and I don't think there's any point in fixing typos there.

**Comment by e-kwsm** [comment]
> Can you undo the changes to mypyc/external/googletest?

Done.

File: test-data/unit/fixtures/object_with_init_subclass.pyi [hunk 1]
```diff
@@ -7,7 +7,7 @@ class object:
 T = TypeVar('T')
 KT = TypeVar('KT')
 VT = TypeVar('VT')
-# copy pased from primitives.pyi
+# copy passed from primitives.pyi
```
Comment (hauntsaninja) [inline test-data/unit/fixtures/object_with_init_subclass.pyi:10]: nit:
```suggestion
# copy pasted from primitives.pyi
```

**Review by hauntsaninja — COMMENTED** [review]
I'm curious what you used to find these?

**Comment by hauntsaninja** [comment]
Thanks!

**Comment by e-kwsm** [comment]
I use https://github.com/bwignall/typochecker.

**Pull request merged**